README.md

```diff
@@ -120,6 +120,12 @@ however, if the metrics request is made using a load balancer or Kubernetes Node
 original request was made might not be the same as the instance's HTTP port. In such a case, the configuration should
 include the `restPort` configuration to tell the exporter which port to use.
 
+## Use the Monitoring Exporter with WebLogic Kubernetes Operator
+
+If you are running operator-managed WebLogic Server domains in Kubernetes, simply add the [`monitoringExporter`](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/domains/Domain.md) configuration element in the domain resource to enable the Monitoring Exporter. For an example, see https://github.com/oracle/weblogic-kubernetes-operator/blob/main/operator/src/test/resources/oracle/kubernetes/weblogic/domain/model/domain-sample-3.yaml.
```

Comment:
A problem with referring to the resource in the operator project is that it is subject to change. I think it will be better to make a simplified configuration that embeds the one in this project.

I can add it to this PR or send one to you.